index.js

```diff
@@ -98,6 +112,10 @@ class ClientServiceBase {
    */
   async registerCertificate() {
     const request = await this._createCertificateRegistrationRequest();
+    return this._registerCertificate(request);
```

Comment:
@feeblefakie As we discussed offline, this change is intended to call from Node.js SDK using serialized binary I/F while keeping the main logic commonized here. See also,
https://github.com/scalar-labs/scalardl-node-client-sdk/pull/39/files#diff-329531d245e95e56f45d166c536ccee5fcd533cc3ca870c2fe5b75483e20e3c3R142.